hypershift-operator/controllers/nodepool/manifests.go

```diff
@@ -69,11 +69,11 @@ func TunedConfigMap(namespace, name string) *corev1.ConfigMap {
 	}
 }
 
-func PerformanceProfileConfigMap(namespace, name string) *corev1.ConfigMap {
+func PerformanceProfileConfigMap(namespace, name, nodePoolName string) *corev1.ConfigMap {
 	return &corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: namespace,
-			Name:      "perfprofile-" + name,
+			Name:      name + "-" + nodePoolName,
```

Comment:
This can overflow the allowed length for a Kubernetes object name. If we require that this name be human-readable, please truncate each section (name, node pool name) and concatenate. If we do not require, please base32(sha224(input)) to get a hash which we can use as the name for this `ConfigMap` and leave the identifying information in labels or annotations.

Reply:
Appreciate your comment. 
`name` and `nodePoolName` are valid qualified DNS subdomain name.
The only problem here is the length as you mentioned. I've fixed it in the last update, thanks!